recipes-debian/libpcap/libpcap_debian.bb

```diff
@@ -0,0 +1,45 @@
+SUMMARY = "Interface for user-level network packet capture"
+DESCRIPTION = "Libpcap provides a portable framework for low-level network \
+monitoring.  Libpcap can provide network statistics collection, \
+security monitoring and network debugging."
+HOMEPAGE = "http://www.tcpdump.org/"
+BUGTRACKER = "http://sourceforge.net/tracker/?group_id=53067&atid=469577"
+SECTION = "libs/network"
+LICENSE = "BSD"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=5eb289217c160e2920d2e35bddc36453 \
+                    file://pcap.h;beginline=1;endline=32;md5=39af3510e011f34b8872f120b1dc31d2"
+DEPENDS = "flex-native bison-native"
+
+inherit debian-package
+require recipes-debian/sources/libpcap.inc
+
+DEBIAN_PATCH_TYPE = 'quilt'
+FILESPATH_append = ":${COREBASE}/meta/recipes-connectivity/libpcap/libpcap"
+
+SRC_URI += "file://0001-pcap-usb-linux.c-add-missing-limits.h-for-musl-syste.patch"
+
+BINCONFIG = "${bindir}/pcap-config"
```

Comment:
Poky's bb file contains [following line ](https://git.yoctoproject.org/cgit.cgi/poky/tree/meta/recipes-connectivity/libpcap/libpcap_1.9.0.bb?h=warrior#n19 ) , so it looks required to build this package.

```
inherit autotools binconfig-disabled pkgconfig bluetooth
```